src/coreclr/vm/peimage.cpp

```diff
@@ -869,6 +876,9 @@ HRESULT PEImage::TryOpenFile(bool takeLock)
         return S_OK;
 
     ErrorModeHolder mode{};
+#if defined(TARGET_ANDROID)
+    m_hFile = GetData(nullptr);
+#else
```

Comment:
```suggestion
```
If the `GetFileHandle` is still called after the other changes. fix the caller instead of this hack?